test/flows/simplified_expenses_checker_flow_test.rb

```diff
@@ -0,0 +1,584 @@
+    end
+
+    should "render text for living on business premises, in the simplified expenses section, if the number of people living on the premises is grater than 0" do
+      add_responses home_or_business_premises_expense?: "live_on_business_premises",
+                    deduct_from_premises?: "40000",
+                    people_live_on_premises?: "2"
+
+      assert_rendered_outcome text: "to deduct from your total business costs - you then claim the balance"
+    end
+
+    should "render text if the user has claimed running costs for the vehicle" do
+      add_responses home_or_business_premises_expense?: "no_expense",
+                    buying_new_vehicle?: "new",
+                    how_much_expect_to_claim?: "50000",
+                    is_vehicle_green?: "low",
+                    price_of_vehicle?: "50",
+                    vehicle_business_use_time?: "50",
+                    drive_business_miles_car_van?: "1000"
+
+      assert_rendered_outcome text: "for the running costs of your car, van or motorcycle"
+    end
+
+    should "render text if the vehicle price exceeds the Capital Allowance write off limit" do
+      add_responses home_or_business_premises_expense?: "no_expense",
+                    buying_new_vehicle?: "new",
+                    how_much_expect_to_claim?: "50000",
+                    is_vehicle_green?: "low",
+                    price_of_vehicle?: "300,000",
+                    vehicle_business_use_time?: "50",
+                    drive_business_miles_car_van?: "1000"
+
+      assert_rendered_outcome text: "The upper limit for Capital Allowance write offs for vans is £250,000"
+    end
+  end
+
+  context "outcome: capital_allowance_result" do
+    setup do
+      testing_node :capital_allowance_result
+      add_responses vehicle_expense?: "car",
+                    home_or_business_premises_expense?: "no_expense",
+                    buying_new_vehicle?: "no",
+                    capital_allowances?: "capital_allowance"
+    end
+
+    should "render text if respondent picked the capital allowances option and did not have any previous expenses" do
+      assert_rendered_outcome text: "You can’t use simplified expenses for your current vehicle because you’ve already claimed capital allowances for it"
+    end
+  end
+
+  context "outcome: you_cant_claim_capital_allowance" do
+    setup do
+      testing_node :you_cant_claim_capital_allowance
+      add_responses vehicle_expense?: "car",
+                    home_or_business_premises_expense?: "no_expense",
+                    buying_new_vehicle?: "no",
+                    capital_allowances?: "simplified_expenses"
+    end
+
+    should "render text if respondent picked the simplified_expenses option and did not have any previous expenses" do
+      assert_rendered_outcome text: "You can’t claim capital allowances for your vehicle because you’ve already claimed simplified expenses for it"
+    end
+  end
+end
```

Comment:
I'd delete these because of the lack of logic. We're already testing that they show up in other bits of this test